.github/merge_rules.json

```diff
@@ -41,6 +41,23 @@
       "approved_by": ["linbinyu", "kit1980", "IvanKobzarev", "dreiss"],
       "mandatory_checks_name": ["Facebook CLA Check", "Lint"]
    },
+   {
+      "name": "Linear Algebra",
+      "patterns": [
+         "aten/src/ATen/native/cuda/linalg/**",
+         "aten/src/ATen/LinalgBackend.h",
+         "aten/src/ATen/native/**/*LinearAlgebra*",
+         "docs/source/linalg.rst",
+         "torch/linalg/**",
+         "torch/_linalg_utils.py",
+         "torch/**/python_linalg_functions.*",
+         "torch/**/linalg.h",
+         "tools/autograd/templates/python_linalg_functions.cpp",
+         "test/test_linalg.py"
+      ],
+      "approved_by": ["nikitaved", "mruberry", "pearu", "Lezcano", "IvanYashchuk"],
```

Comment:
@mruberry is there a github team that's representative of this group? I'd prefer we use github teams rather than individual usernames if possible

Reply:
No but we can always create one

Reply:
We tried doing this, and we actually **cannot** given the tools today. Even with admin access, @suo couldn't create the teams externally + had to go through the internal rules, and then the internal rule doesn't allow adding external contributors to orgs (they can only be added to repos). 

I think we can resolve this with the OSS team a different time and land this change if there are no other blockers.